### PR TITLE
SSCS-10321 Fixes

### PR DESCRIPTION
**Pull request by TeaWithLucas**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SSCS-10321

### Change description ###

- [x] Fix ExUi Url
- [x] Fix Sscs Service Code

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by dattipoeHMCTS — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/model/HearingWrapper.java [hunk 1]
```diff
@@ -14,4 +14,6 @@
 public class HearingWrapper {
     private SscsCaseData caseData;
     private HearingState state;
+    private String sscsServiceCode;
+    private String exUiUrl;
```
Comment (wmarshid) [inline src/main/java/uk/gov/hmcts/reform/sscs/model/HearingWrapper.java:18]: Why are we passing a url in the hearingWrapper now?

**Review by nickwest1 — CHANGES_REQUESTED** [review]

**Review by wmarshid — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/model/HearingWrapper.java (hunk 1 above)
Comment (wmarshid) [reply src/main/java/uk/gov/hmcts/reform/sscs/model/HearingWrapper.java:18]: Discussed with @TeaWithLucas happy with this approach

File: src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsCaseMapping.java [hunk 2]
```diff
@@ -57,16 +42,16 @@ public static CaseDetails buildHearingCaseDetails(HearingWrapper wrapper) {
...
     }
 
     public static String getCaseID(SscsCaseData caseData) {
         return caseData.getCcdCaseId();
     }
 
-    public static String getCaseDeepLink(SscsCaseData caseData) {
-        return String.format("%s/cases/case-details/%s", exUiUrl, getCaseID(caseData));
+    public static String getCaseDeepLink(HearingWrapper wrapper) {
+        return String.format("%s/cases/case-details/%s", wrapper.getExUiUrl(), getCaseID(wrapper.getCaseData()));
```
[the remark this answers is not in the document]
Comment (TeaWithLucas) [reply src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsCaseMapping.java:54]: Sorted

**Review by dattipoeHMCTS — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsCaseMappingTest.java [hunk 3]
```diff
@@ -97,7 +97,7 @@ void getCaseDeepLink() {
                 .exUiUrl(EX_UI_URL)
                 .build();
         String result = HearingsCaseMapping.getCaseDeepLink(wrapper);
-        String expected = String.format("%s/cases/case-details/%s", EX_UI_URL, CASE_ID);
+        String expected = String.format(HearingsCaseMapping.CASE_DETAILS_URL, EX_UI_URL, CASE_ID);
```
Comment (wmarshid) [inline src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsCaseMappingTest.java:100]: Optional: static import this please

**Review by wmarshid — APPROVED** [review]

**Pull request merged**